### ISSUE TITLE
What is the type of loss in the log?

### ISSUE THREAD

**Issue by Wang-Lin-boop** [microsoft/FLAML#655]
When I trained a model with the following parameters, FLAML gave me some log information, but I still had some confusion about two parameters, ```loss``` and ```eval_loss```.

My params:
``` python
    automl_settings = {
        "n_jobs": 2, 
        "verbose":2,
        "n_concurrent_trials" : 5,
        "gpu_per_trial": 0,
        "max_iter": 100,
        "min_sample_size": 50000, 
        "task": "seq-regression",
        "ensemble": True,
        "log_file_name": f"{model_name}.log",
    }
    automl_settings["fit_kwargs_by_estimator"] = {  # setting the huggingface arguments
        "transformer": {
            "output_dir": model_name, 
            "fp16": False,
        }
    }
    automl_settings['custom_hp'] = {
        "transformer": {
            "model_path": {
                "init_value": "google/electra-small-discriminator",
                "domain": tune.choice(["albert-base-v2", "google/electra-small-discriminator", "facebook/muppet-roberta-base"]),
            },
            "learning_rate": {
                "domain": tune.choice([1e-4, 1e-5]),
            },
            "num_train_epochs": {
                "domain": tune.randint(1, 15)
            }
        }
    }
    automl_settings['metric'] = "r2"
    seq_predictor.fit(
        X_train=X_train, y_train=y_train, X_val=X_val, y_val=y_val, **automl_settings
    )
```
Output:
```
{'eval_loss': 2.2773654460906982, 'eval_automl_metric': 0.8865896318335299, 'eval_runtime': 585.1213, 'eval_samples_per_second': 19.003, 'eval_steps_per_second': 19.003, 'epoch': 3.0}
{'train_runtime': 4376.1113, 'train_samples_per_second': 6.855, 'train_steps_per_second': 0.215, 'train_loss': 3.975694877778887, 'epoch': 3.0}
```
I'm guessing that ```eval_automl_metric``` means that the metric I specified is also R2, but I don't find an option in the documentation to specify the loss function in the ```fit``` function, is the default loss RMSE?

**Comment by liususan091219** [comment]
Hi! eval_loss is the loss (e.g., cross entropy loss) of the validation data, whereas eval_automl_metric is 1-metric (of the validation data). Yes, in your case, the metric is set to R2, so eval_automl_metric = 1- the R2 score. 

When you set the task to seq-regression, the default metric is R2: https://github.com/microsoft/FLAML/blob/main/flaml/nlp/utils.py#L18

If you want to use RMSE, you can set metric by:

```python
automl_settings['metric'] = "rmse"
```

**Comment by Wang-Lin-boop** [comment]
Thank you for your reply.
I'm doing a regression task where two sequences determine a value, in fact I tried AutoGluon and got very poor results, probably it's a difficult task (for a novice like me), emm...... let me try again with other parameters.

**Comment by Wang-Lin-boop** [comment]
Thank you for your reply.
I'm doing a regression task where two sequences determine a value, in fact I tried AutoGluon and got very poor results, probably it's a difficult task (for a novice like me), emm...... let me try again with other parameters.

**Comment by Wang-Lin-boop** [comment]
@liususan091219  An another topic, can I specify to train two different transformer models separately for two different sentences in the sequence regression task? 

After all, while in more common NLP tasks two sentences are two similar languages, in my task the two sentences are in almost completely different languages, and I wanted to capture the connection between them.

**Comment by liususan091219** [comment]
Hello, thanks for sharing your use case. It may be helpful to learn more details about your application and I will try my best to help you improve your results. We invited you to gitters, feel free to drop me a line there!

**Comment by Wang-Lin-boop** [comment]
Thanks, I will try to complete a whole training session with the default FLAML settings first. I will share my case with you in detail by email (or other ways you like). BTW, I have just signed up for gitters, maybe I won't get your invitation before then.

**Comment by liususan091219** [comment]
> Thanks, I will try to complete a whole training session with the default FLAML settings first. I will share my case with you in detail by email (or other ways you like). BTW, I have just signed up for gitters, maybe I won't get your invitation before then.

Sounds good. My email address is xliu127@stevens.edu, and I just added you to Gitter FLAMLer through your github account!
